stingray/lightcurve.py

```diff
@@ -1268,17 +1372,14 @@ def write(self, filename, format_='pickle', **kwargs):
         format\_: str
             Available options are 'pickle', 'hdf5', 'ascii'
         """
-
+        _ = self.counts, self.counts_err, self.countrate, self.countrate_err
```

Comment:
Is there a reason why this needs to be evaluated here, rather than in the `io.write` call for `ascii` below. Presumably, if saved in binary form (e.g. pickle), it's possible to still lazily evaluate these later once you re-read the object from file, so there's no point explicitly storing them?

Reply:
`pickle` is the only one where it's not strictly needed. However, I think that this read-write infrastructure needs a big revamp (not in this PR, I think it will come naturally when we discuss the new memory-mapping infrastructure.) and it doesn't look super useful in the current state, so I kept it simple.
What do you think?

Reply:
yeah, agreed. Let's go for this now and continue to chat about the read/write infrastructure. :)